src/pagination/pagination.component.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.pagination.component:pfPagination
+ * @restrict E
+ *
+ * @param {number} pageNumber The current page number
+ * @param {number} numTotalItems The total number of items in the data set.  When a filter is applied, update the <code>numTotalItems</code>
+ * accordingly.
+ * @param {number[]} pageSizeIncrements (optional) Page size increments for the 'per page' dropdown.  If not
+ * specified, the default values are: [5, 10, 20, 40, 80, 100]
+ * @param {number} pageSize (optional) The initial page size to use.  If not specified, the default will be
+ * the first increment in the <code>pageSizeIncrements</code> array.
+ * @description
+ * Component for pagination controls used in various views (list, card, table)
+ *
+ * @example
+ <example module="patternfly.pagination">
+
+ <file name="index.html">
+   <div ng-controller="PageCtrl">
+     <div class="col-md-12">
+       <div ng-repeat="item in items | startFrom:(pageNumber - 1)*pageSize | limitTo:pageSize" class="col-md-12">
+         <div class="row">
+           <div class="col-md-3">
+             <span>{{item.id}}</span>
+           </div>
+           <div class="col-md-7">
+             <span>{{item.status}}</span>
+           </div>
+           <div class="col-md-2">
+             <span>{{item.value}}</span>
+           </div>
+         </div>
+       </div>
+     </div>
+     <pf-pagination
+         page-size="pageSize",
+         page-number="pageNumber"
+         num-total-items="numTotalItems">
+     </pf-pagination>
+   </div>
+ </file>
+ <file name="script.js">
+ angular.module( 'patternfly.pagination').controller( 'PageCtrl', function( $scope, PaginationService ) {
+   $scope.pageSize = 10;
+   $scope.pageNumber = 1;
+
+   $scope.items = [];
+   for(i = 1; i <= 126; i++) {
+      $scope.items.push({id: i, status: 'Ok', value: Math.floor(Math.random() * (1000 - 1 + 1)) + 1});
+   }
+
+   $scope.numTotalItems = $scope.items.length;
+ });
+ </file>
+ </example>
+ */
+angular.module('patternfly.pagination').component('pfPagination', {
+  transclude: true,
+  templateUrl: 'pagination/pagination.html',
+  bindings: {
+    pageNumber: '=',
+    numTotalItems: "<",
+    pageSizeIncrements: '<?',
+    pageSize: "=?"
+  },
+  controller: function ($scope, $log) {
+    'use strict';
+    var ctrl = this;
+
+    var defaultPageSizeIncrements = [5, 10, 20, 40, 80, 100];
```

Comment:
Ah yes, not so clear. Had to do with me reducing the number of table entries to "2" then still seeing the page results dropdown showing increments up to 100.